htdocs/holiday/class/holiday.class.php

```diff
@@ -259,6 +259,17 @@ public function create($user, $notrigger = 0)
 			$this->error = "ErrorBadParameterFkType"; return -1;
 		}
 
+		$checkBalance = getDictvalue(MAIN_DB_PREFIX.'c_holiday_types', 'block_if_negative', $this->fk_type);
+
+		if ($checkBalance > 0) {
```

Comment:
We must check when we validate only. 
But when we create, we must check only if status is not draft.